scripts/travis_script

```diff
@@ -41,6 +42,10 @@ case "$TASK" in
         # https://docs.travis-ci.com/user/reference/osx/#JDK-and-OS-X
         ./gradlew clean assemble --stacktrace
         ./gradlew check
+        java_version=`java -version 2>&1 | awk -F '"' '/version/ {print $2}'`
+        if [[ ! "$java_version" < "1.8" ]]; then
```

Comment:
I don't think this check is necessary since we are currently only testing with Java 8 on OS X.

Reply:
Done.